vuepress/docs/v6.3.2/tutorials/samples/landing-page.md

```diff
@@ -4,36 +4,49 @@ sidebarDepth: 2
 
-## Solution Template Details
+## Entando Customer Portal
 
-Specific features and use cases are highlighted below:
+The Entando Customer Portal streamlines development of a customer-facing, self-service subscription management application. The platform integrates RBAC user supervision and Jira Service Management as a ticket tracking system.
 
-- **Entando Customer Portal**:\
-Streamlines development of a customer-facing subscription management platform for a modern self-service experience. The framework consists of Jira Service Management integration for ticketing and tracking, RBAC identity supervision, and integrated management of subscriptions, projects and users.
+<img src="./landing-images/customer-portal.png" width="533" height="240.49">
 
-- [**Entando Hub**](hub.md):\
-Installing a Hub instance empowers a team to share and collaborate on proprietary or Entando Open Source components. Capabilities, versions and metadata can be transparently analyzed, managed and published.
+## Entando Hub
 
-- [**Entando Process Driven Applications Plugin**](pda-tutorial.md):\
-Provides a comprehensive and versatile automation scheme to manage and complete business processes. Resources and mechanisms comprise a custom UX layer, a Spring Boot Backend for Frontend microservice interface, and integration with the [Red Hat Process Automation Manager](https://www.redhat.com/en/technologies/jboss-middleware/process-automation-manager).
+The [Entando Hub](hub.md) empowers a team to share and collaborate on proprietary or Entando Open Source components. Capabilities, versions and metadata can be transparently analyzed, managed and published.
 
-- [**Entando Standard Banking Demo**](install-standard-demo.md):\
-Showcases modularity benefits and composability innovation as implemented by the Entando platform. Prominent features include RBAC Keycloak integration, coexistence of React and Angular MFEs, pluggable Spring Boot microservices, and sophisticated MFE communication techniques.
+<img src="./landing-images/hub-v1.png" width="533" height="465.02">
 
+## Entando Process Driven Applications Plugin
+
+The [Entando Process Driven Applications Plugin](pda-tutorial.md) provides a comprehensive and versatile automation scheme for Business Process Management. It comprises a custom UX layer, a Spring Boot Backend for Frontend microservice interface, and integration with the [Red Hat Process Automation Manager](https://www.redhat.com/en/technologies/jboss-middleware/process-automation-manager).
+
+<img src="./landing-images/task-list.png" width="533" height="368.34">
+
+## Entando Standard Banking Demo
+
+The [Entando Standard Banking Demo](install-standard-demo.md) showcases modularity benefits and composability innovation as implemented by the Entando platform. Prominent features include RBAC Keycloak integration, coexistence of React and Angular MFEs, pluggable Spring Boot microservices, and sophisticated MFE communication techniques.
```

Comment:
The writing is great here, straightforward and concise but for the features, can we mention some of the widgets listed in the details to exemplify what it can do?

Reply:
we can definitely do that! i'm going to try to reword this a bit more attractively, focusing on clarity and benefits, but i'm a little worried about deviating from short descriptions. i was thinking this (and the pda blurb) made for enough bait to entice the appropriate audience, but we can try to find better language